src/server/system_services/pool_server.js

```diff
@@ -834,6 +834,16 @@ async function get_cloud_services_stats(req) {
     ]);
 
     const all_services = {};
+    let services_types_arr = ['AWS', 'AZURE', 'S3_COMPATIBLE', 'GOOGLE', 'FLASHBLADE', 'NET_STORAGE'];
```

Comment:
We have lots of places where we declare this array (APIs, schemas, code paths).
Not all are using exactly this array as is, some are using a subarray. 
I think that most of the APIs and schemas should have the same definition in common_api.
Regarding the code paths, we can also aggregate that to a single point in config.js for example.
Obviously, the paths that need subarray will need to declare it locally.

Reply:
Better to declare as const instead of let in this case (this is a constant array that won't be changed).
Even if you will decide to not change it.

Reply:
Since now you are sending all of the services we need to make sure what to show and not show in the UI. 
I think there should be filtering in the UI side.

Reply:
Regardless of the location of it (config.js sounds good to me) keep in mind that `const` only affects the ability to change the reference to the array, but in order to make the array itself immutable you should also freeze the dude:
```
const services_types_arr = Object.freeze(['AWS', 'AZURE', 'S3_COMPATIBLE', 'GOOGLE', 'FLASHBLADE', 'NET_STORAGE']);
```